packages/portfolio-deploy/src/contract-control.js

```diff
@@ -312,8 +332,27 @@ export const prepareContractControl = (zone, svcs) => {
         trace('done');
         return qty;
       },
+
+      revoke() {
+        const { name, kit, revoked } = this.state;
+        trace(name, 'revoke', kit?.adminFacet);
+        !revoked || Fail`revoked`;
+
+        trace(name, 'revoked');
+        this.state.revoked = true;
```

Comment:
Should this also break references? Clear the kit?
```
      this.state.kit = undefined;
```

Reply:
not needed since everything is guarded on revocation, but we could.